client/src/main/java/org/eclipse/hono/client/HonoClient.java

```diff
@@ -338,6 +338,49 @@
     Future<MessageConsumer> createEventConsumer(String tenantId, BiConsumer<ProtonDelivery, Message> eventConsumer,
             Handler<Void> closeHandler);
 
+    Future<MessageConsumer> createCommandResponseConsumer(String tenantId, String replyId, Consumer<Message> commandResponseConsumer,
+            Handler<Void> closeHandler);
+
+    Future<MessageConsumer> createCommandResponseConsumer(String tenantId, String replyId,
+            BiConsumer<ProtonDelivery, Message> commandResponseConsumer,
+            Handler<Void> closeHandler);
+
+    /**
+     * Creates a client for consuming async command responses from Hono's north bound <em>Command API</em>.
+     * <p>
+     * The command responses passed in to the responses consumer will be settled automatically if the consumer does not
```

Comment:
> The command responses passed in to the ~~responses~~ consumer will be settled automatically if the consumer does not throw an exception ~~and does not manually handle the message disposition using the passed in delivery~~.

There is no delivery being passed in ...

Reply:
I fixed this.